feconf.py

```diff
@@ -97,12 +97,11 @@
     'html_field_types_to_rule_specs.json')
 
 # A mapping of interaction ids to classifier properties.
-# As of now we support only one interaction per interaction however, we do have
-# necessary storage infrastructure to support multiple algorithms per
-# interaction. Hence, whenever we find a secondary algorithm candidate for any
-# of the supported interactions, the logical functions to support multiple
-# algorithms needs to be implemented.
-# TODO(@prasanna08): More info: https://github.com/oppia/oppia/issues/10217
+# TODO(#10217): As of now we support only one interaction per interaction
+# however, we do have necessary storage infrastructure to support multiple
```

Comment:
... per interaction. However...

necessary --> the necessary

Reply:
Done